interfaces/Portal/src/app/pages/payment/payment.page.ts

```diff
@@ -14,8 +14,7 @@ export class PaymentPage {
   public Permission = Permission;
 
   public programId = this.route.snapshot.params.id;
-  public thisPhase = ProgramPhase.payment;
-  public isReady: boolean;
+  public thisPhase = ProgramTab.payment;
```

Comment:
Are there any uses from 'outside'? So maybe we can turn this into a `private` property? Or possibly remove/rename it?